nixos/modules/config/no-x-libs.nix

```diff
@@ -9,7 +9,7 @@ with lib;
   options = {
     environment.noXlibs = mkOption {
       type = types.bool;
-      default = false;
+      default = !config.services.xserver.enable && !config.services.xrdp.enable;
```

Comment:
I thought about ```environment.noXlibs``` exactly as the setting which should cause rebuilds in order to make closures with less dependencies.
How would it be possible otherwise to get rid on the X11 dependencies without rebuilding the closures which have them ?

Reply:
I agree having hydra build X-less variants would be a good solution to this. Are there any obstacles in the way other than just doing the work?